lib/ueberauth/strategy/amco/plugs/authenticated_user.ex

```diff
@@ -18,12 +18,16 @@ defmodule Ueberauth.Strategy.Amco.Plugs.AuthenticatedUser do
       raise "error_handler option is required."
     end
 
+    unless Keyword.get(opts, :access_token_source) do
+      raise "access_token_source option is required."
```

Comment:
IMO we should start using custom exceptions, instead of plain messages, it helps a lot to devs when debugging:

https://vitalflux.com/4-reasons-need-create-customs-exceptions/#:~:text=Improved%20Exception%20Handling%3A%20Custom%20exceptions,a%20miss%20if%20done%20religiously.

Some opinions I share ☝️ 

on elixir you use the `defexception` on a `MyException` module to set them